FacebookEventForwarder.js

```diff
@@ -122,6 +122,9 @@ var mpFacebookKit = (function (exports) {
                       if (event.CurrencyCode) {
                           params['currency'] = event.CurrencyCode;
                       }
+                      else{
+                          params['currency'] = 'USD'; // Inject CurrencyCode if it doesn't exist

```

Comment:
Is it possible for this to be an empty string or `null`? I worry that we don't want to assume USD for foreign clients